### DISCUSSION TITLE
Why are Essential assets children of SettledContract?

### DISCUSSION THREAD

**Discussion by jpmckinney** [OP-TED/eForms-SDK#250] [Q&A] [answered]
In the old T02 form, II.4) Essential assets is in section II, which describes the object of procurement. These fields were singular (at most one per notice).

In eForms, the corresponding fields (OPP-020 to 023) are children of SettledContract. That makes the fields plural. There also seems to be a semantic shift, since SettledContract corresponds to section V, which describes the award of contract.

I just want to confirm whether (1) these fields can in reality hold different values for different contracts? (I know the schema permits it, but does this actually correspond to real procedures?) and (2) that these fields are, semantically, about the contract, not about the object of procurement? (I know the schema makes them about the contract, but is that correct?)

**Comment by rouschr** [comment]
Maybe @pdonohoe can help you out as he knows about both the old and the new forms.

**Reply by jpmckinney** [reply]
@pdonohoe What do you think?

**Comment by YvesJo** [comment, marked as answer]
Hi James,
This is a purely business related question that should also be addressed with business people, therefore I invite you to address it to the Helpdesk instead.
Thanks for your understanding.
